src/app/filter-gender/filter-gender.component.scss

```diff
@@ -0,0 +1,7 @@
+button {
```

Comment:
Please add a class to the button and assign the styles to the class instead of tag.